compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -6146,6 +6146,74 @@ TR::Register *commonLoadEvaluator(TR::Node *node, TR::InstOpCode::Mnemonic op, i
    return commonLoadEvaluator(node, op, size, tempReg, cg);
    }
 
+static TR::Register *
+loadAddress(TR::Node *node, TR::MemoryReference *mr, TR::CodeGenerator *cg)
```

Comment:
I think it is more appropriate to call this function `computeAddress` or `loadEffectiveAddress` (to borrow an Intel term) because it doesn't actually perform any memory load (as its name implies).